coalib/parsing/LineParser.py

```diff
@@ -62,8 +62,16 @@ def parse(self, line):
         # Escapes in value might be needed by the bears
         keys, value = self.__extract_keys_and_value(line)
 
+        all_delimiters = self.key_value_delimiters
+        all_delimiters += self.key_delimiters
+        all_delimiters += self.comment_seperators
+        all_delimiters += self.section_override_delimiters
+        all_delimiters = "".join(all_delimiters)
```

Comment:
No, all four are tuples. So we can set `all_delimiters` to be `self.key_value_delimiters` first and then when we do `+= self.key_delimiters` next. This will create a new tuple concatenating `all_delimiters` and `self.key_delimiters`. And similarly the rest. So after line 68 `all_delimiters` is a tuple and then we stringify it next.